### DISCUSSION TITLE
Created a design in Figma with UI Kit. Now would like to export the design to code (css/html/ts(x)) - how can I do that?

### DISCUSSION THREAD

**Discussion by piprog** [primefaces/community#2055] [Figma UI Kit]
Hi Team,

The design is of a full screen, completely done with components from UI Kit. Of course, we can do a manual copy-paste, but would make much more sense to have at least a rough export that can be further tuned.

Any help much appreciated

**Comment by onursenture** [comment]
Hi, Unfortunately, we currently do not have such a service. We have plans to make things easier when transitioning from design to code. However, these plans are not aimed at converting a design made with UI Kit directly into code, even roughly. Instead, they will help you transfer the theme changes you make in the UI Kit to the theme you use in the component library.
